src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -2,11 +2,10 @@
 import errno
 import logging
 
-from ceph.deployment.service_spec import NFSServiceSpec, PlacementSpec
```

Comment:
@varshar16 could you note down in the commit message why reordering the imports helps to address the issue?

Reply:
Done.